languages/common-lisp/exercises/concept/lists/.meta/design.md

```diff
@@ -0,0 +1,66 @@
+This issue describes how to implement the `lists` concept exercise for the Common Lisp track.
+
+## Getting started
```

Comment:
I think I've been chopping out larges parts of the issues when I paste them here. I've just been leaving Learning, Scope, Concepts, and Prerequisites, I think. I can see a case for both approaches though, so whatever you think is best :)